packages/main/src/themes/Menu.css

```diff
@@ -86,4 +110,10 @@
 .ui5-menu-rp[sub-menu] {
 	margin-top: 0.25rem;
 	margin-inline: var(--_ui5_menu_submenu_margin_offset);
+}
+
+.ui5-menu-item::part(additional-text) {
+	margin: 0 0 0 1rem;
```

Comment:
here we introduce asymmetric margin - does it work properly in RTL - I suggest using css logical props here as well - probably [margin-inline-start](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline-start)

Reply:
Done.